src/sql/platform/accounts/common/interfaces.ts

```diff
@@ -58,7 +58,8 @@ export enum AzureResource {
 	MicrosoftResourceManagement = 5,
 	AzureDevOps = 6,
 	MsGraph = 7,
-	AzureLogAnalytics = 8
+	AzureLogAnalytics = 8,
```

Comment:
you may also add `AzureStorage = 9` here just to keep the enums in sync in the various places.

Reply:
sounds good, I have added AzureStorage too